tests/e2e/types/test_client.go

```diff
@@ -105,40 +105,29 @@ func (p *TestConsumerClient) BootstrapContracts() (*ConsumerContract, error) {
 	btcFinalityContractWasmId := p.Chain.StoreCodeFile("../testdata/btc_finality.wasm").CodeID
 
 	// Instantiate the contract
-	// TODO: parameterise
-	btcStakingInitMsg := map[string]interface{}{
-		"admin": p.GetSender().String(),
-	}
-	btcStakingInitMsgBytes, err := json.Marshal(btcStakingInitMsg)
+	// instantiate Babylon contract using cli.Parse approach
+	msgInit, err := cli.ParseInstantiateArgs(
```

Comment:
Nice. Next step IMO would be to use this instantiation method in the other e2e tests as well.

Reply:
Correct, once we have a new release of babylon-contract, we can update the CLI to include the reward stuff and have everything in e2e